README.md

```diff
@@ -68,7 +85,9 @@ docker build -t firecloud-orch .
 Run the container:
 ```
 docker run --name orch \
+  -p 8080:8080
```

Comment:
This surprises me. I thought the "EXPOSE 8080" in the Dockerfile took care of this. Is that not true? If not, then I don't even know what the EXPOSE line is supposed to do. So much to learn.